### PR TITLE
Implemented NotificationView

### PR DESCRIPTION
**Pull request by drewsetski**
## Description
I've created new pull request because I had to merge with last changes on main repository and there were conflicts #9160

## Fast guide 
```
kd.singletons.notificationViewController.addNotification
     type : ['success', 'update', 'warning', 'caution']
     duration : integer (ms)
     dismissible : boolean
     content : string
     primaryButtonTitle : string
     onPrimaryButtonClick : function
     secondaryButtonTitle : string
     onSecondaryButtonClick : function
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by drewsetski** [comment]
> Video: https://www.dropbox.com/s/y8q3an8xgoquc0p/notificationsViewController.mov?dl=0

**Review by usirin — CHANGES_REQUESTED** [review]

**Comment by drewsetski** [comment]
I've pushed little fix to conditions, but it hadn't appear on github...

File: client/component-lab/Notification/NotificationView.coffee
Comment (hakankaradis) [inline client/component-lab/Notification/NotificationView.coffee]: `@isMounted` is a special function in Reactjs. It throws some warning, [check from here](https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html)

**Review by hakankaradis — CHANGES_REQUESTED** [review]
Can you please check once again the styles.

It doesn't work properly.
Thanks

**Comment by gokmen** [comment]
what is the latest status about this? cc/ @usirin @hakankaradis

**Review by hakankaradis — APPROVED** [review]

**Comment by hakankaradis** [comment]
@gokmen I believe it is ready to go.
After @usirin  approves the changes, I will pull the branch and open new pull request to run our tests
That means we will not merge this pr.

**Comment by hakankaradis** [comment]
@gokmen, @usirin pr is reopened  #10218, fyi

**Pull request closed without merging**
